mobile/src/main/java/com/github/shadowsocks/ProfileConfigFragment.kt

```diff
@@ -102,18 +103,6 @@ class ProfileConfigFragment : PreferenceFragmentCompat(),
         findPreference<Preference>(Key.udpdns)!!.isEnabled = serviceMode != Key.modeProxy
         plugin = findPreference(Key.plugin)!!
         pluginConfigure = findPreference(Key.pluginConfigure)!!
-        plugin.unknownValueSummary = getString(R.string.plugin_unknown)
-        plugin.setOnPreferenceChangeListener { _, newValue ->
```

Comment:
Why removed `setOnPreferenceChangeListener` here?

Reply:
Hmm it is supposed to be handled by `onActivityResult` now.

Reply:
Do you where the plugin icon and name updated when the plugin is switch in the dialog? I'm debugging the plugin switching issue.

Reply:
Committed a fix via web editor. Let me know if it works. :)